packages/amplication-client/src/CommandPalette/CommandPalette.tsx

```diff
@@ -101,21 +102,27 @@ type Props = {
 };
 
 const CommandPalette = ({ trigger }: Props) => {
-  const match = useRouteMatch<{ resourceId: string }>("/:resourceId/");
+  const { currentWorkspace, currentProject, currentResource } = useContext(
+    AppContext
+  );
 
-  const { resourceId } = match?.params || {};
+  const projectBaseUrl = useMemo(
+    () => `/${currentWorkspace?.id}/${currentProject?.id}`,
+    [currentWorkspace, currentProject]
+  );
 
   const history = useHistory();
   const [query, setQuery] = useState("");
   const handleChange = (inputValue: string, userQuery: string) => {
     setQuery(userQuery);
   };
   const { data } = useQuery<TData>(SEARCH, {
-    variables: { query },
+    variables: { query, projectId: currentProject?.id },
```

Comment:
do we need to add skip ?

Reply:
no, it brings all data when it first opens